ting_openformat.field.inc

```diff
@@ -529,6 +533,53 @@ function _ting_openformat_parse_analytic_content_element($element) {
 
 }
 
+/**
+ * Helper function to parse contributors
+ * @param $contributors
+ * @param null $entity
+ * @param array $options
+ * @return array|null
+ */
+function _ting_openformat_parse_contributors($contributors) {
+  if (!isset($contributors)) {
+    return null;
+  }
+  $render_array = array();
+  // Parse through elements
+  if(gettype($contributors["contributors"]) === string) {
```

Comment:
Overvej evt. is_string i stedet for. Den er mere simpel. Derudover burde string være "string" da metoden returnere en streng.